app/models/service_template_ansible_playbook.rb

```diff
@@ -93,6 +93,7 @@ def self.build_parameter_list(name, description, info)
       :playbook                 => playbook.name,
       :inventory                => tower.provider.default_inventory,
       :become_enabled           => info[:become_enabled].present?,
+      :verbosity                => info[:verbosity] || 0,
```

Comment:
@syncrou Will this always be passed as an integer value?  If not should we be using `.presence` here to protect against an possible empty string?